Update password-store extension

TimShilov opened the pull request:
## Description

<!-- A summary of your change. If you add a new extension or command, explain what it does. -->
This PR adds an ability to specify a additional custom `PATH` values.

For me the extension wasn't working because I use Nix Home Manager to install packages and in my case the `pass` executable is located in `/etc/profiles/per-user/<username>/bin/pass` and extension was unable to find it.

## Screencast

<!-- If you add a new extension or command, include a screencast (or screenshot for straightforward changes). A good screencast will make the review much faster - especially if your extension requires registration in other services.  -->

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [ ] I checked that files in the `assets` folder are used by the extension itself
- [ ] I checked that assets used by the `README` are placed outside of the `metadata` folder

pernielsentikaer commented on extensions/password-store/src/utils/cmd.util.ts:
```diff
@@ -1,5 +1,7 @@
+import { getPreferenceValues } from "@raycast/api";
 import { exec } from "child_process";
 import { promisify } from "node:util";
+import { Preferences } from "../interfaces";
```
You don't need to manually set preference types as this is autogenerated in `raycast-env.d.ts`

Can we remove and rely on the auto generated?

pernielsentikaer commented on extensions/password-store/src/interfaces/preferences.interface.ts:
```diff
@@ -1,4 +1,5 @@
 export interface Preferences {
```
Remove this file

pernielsentikaer commented:
Could you check my comments @TimShilov 😊

TimShilov commented:
@pernielsentikaer Thanks for review. I have removed the custom `Preferences` interface.
Also, while at it, I have also fixed a few typos in `README`... which caused `prettier` to reformat the whole file causing bigger diff than it should be. 😬

pernielsentikaer reviewed (approved):
Hi 👋 

Looks good to me, approved 🔥

TimShilov commented:
@pernielsentikaer Sorry for jumping in after the approval, but I have a general question.
I just noticed that the `NPM Check` mentions that there are a ton of deprecated and outdated depedencies. This is just the second time I contribute to Raycast extensions, so I wonder what's the Raycast's policy on the dependency updates?
Is it okay if I just update then as part of this feature PR? Should the new PR be created just for the updates? Or maybe the dependencies are managed centralized somehow? 🤔 
I'm happy to update them, just not sure if it's okay to include that in the same PR. I usually prefer PRs to be atomic and focus on just one thing.
